### PR TITLE
Add support for docutils 0.14

### PR DESCRIPTION
**Pull request by tnir**
Subject: Add docutils 0.14 to test matrix

### Feature or Bugfix
- Testing

### Purpose
- Add docutils 0.14 to CI test matrix as docutils 0.14 is just out.

### Relates
- http://docutils.sourceforge.net/RELEASE-NOTES.html#release-0-14-2017-08-03
- https://github.com/sphinx-doc/sphinx/pull/3217#discussion_r91833953
  - the latest two versions are to be used for CI tests on Travis CI (except tox.ini)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,20 +18,20 @@ env:
     - PYTHONWARNINGS=all
     - SKIP_LATEX_BUILD=1
   matrix:
-    - DOCUTILS=0.12
```
Comment (tk0miya) [inline .travis.yml:21]: why dropping this?

Comment (tnir) [reply .travis.yml:21]: I just followed their own decision two owners (seems including you) made to keep the latest two versions to be used for tests as described below:

https://github.com/sphinx-doc/sphinx/pull/4050#issue-256475395

> - https://github.com/sphinx-doc/sphinx/pull/3217#discussion_r91833953
>  - the latest two versions are to be used for CI tests on Travis CI (except tox.ini)

Comment (tk0miya) [reply .travis.yml:21]: Oh, sorry. I'd completely forgotten it.
Okay, let's go forward :-)

**Comment by tk0miya** [comment]
Thanks!

**Pull request merged**